ci/sse-infrastructure.yml

```diff
@@ -0,0 +1,39 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Resources:
+  SelfServiceFrontECRRepository:
+    Type: AWS::ECR::Repository
+    Properties:
+      RepositoryName: self-service/frontend
+      ImageTagMutability: IMMUTABLE
+      ImageScanningConfiguration:
+        ScanOnPush: true
+      EncryptionConfiguration:
+        EncryptionType: KMS
+      Tags:
+        - Key: "application"
+          Value: "di-self-service"
+      LifecyclePolicy:
+        RegistryId: !Ref AWS::AccountId
+        LifecyclePolicyText: '{
```

Comment:
So this deletes images older than five days?  Will this cause us any problems - I seem to remember hub key rotation crapping out at some point because images were deleted.  It makes perfect sense for previews but is it what we want for our production image?

Reply:
This is only used for previews, the process for prod will be different. And if the image gets deleted and we need it again, the pipeline will just build and push it again.

Reply:
Think of it as a deplouyment artifact - once the app is deployed, we can trash it. Ideally we'd keep one image per branch until the deployment for the branch is deleted but this is probably too much effort that it's worth. The image can be easily re-built if needed instead.